test/controllers/api/v1/code_schools_controller_test.rb

```diff
@@ -14,25 +20,41 @@ class Api::V1::CodeSchoolsControllerTest < ActionDispatch::IntegrationTest
         name: "CoderSchool"
       }
     }
-    post api_v1_code_schools_url, params: params, as: :json
+    post api_v1_code_schools_url,
+      params: params,
+      headers: @headers,
+      as: :json
 
-    assert JSON.parse(response.body)["errors"].include? "Url can't be blank"
+    errors = JSON.parse(response.body)["errors"]
+    assert errors.include? "Url can't be blank"
   end
 
   test ":index endpoint returns a JSON list of all CodeSchools" do
     get api_v1_code_schools_path, as: :json
 
-    assert_equal JSON.parse(response.body)[0]["name"], "CoderSchool"
-    assert_not_nil JSON.parse(response.body)[0]["locations"]
-    assert_not_nil JSON.parse(response.body)[0]["locations"].first["address1"]
-    assert_equal JSON.parse(response.body)[0]["locations"].first["address1"], "2405 Nugget Lane"
+    body = JSON.parse(response.body)[0]
+    locations = body["locations"]
+    assert_equal body["name"], "CoderSchool"
+    assert_not_nil locations
+    assert_not_nil locations.first["address1"]
+    assert_equal locations.first["address1"], "2405 Nugget Lane"
   end
 
   test ":create endpoint creates a CodeSchool successfully" do
-    post api_v1_code_schools_path(@school), params: {code_school: @school}, as: :json
+    post api_v1_code_schools_path(@school),
+      params: {code_school: @school},
```

Comment:
Small nit, can you update the padding on all of these hashes around the `{ }`. i.e.

```ruby
{ code_school: @school }
```